packages/composites/historical-average/src/dataProvider.ts

```diff
@@ -23,12 +23,16 @@ export const getPriceProvider =
       }
     } catch (error) {
       Logger.error(`Request to ${source} adapter failed: ${error}`)
-      throw new Error(
-        `Failed to request the ${source} adapter. Ensure that the ${source.toUpperCase()}_ADAPTER_URL environment variable is correctly pointed to the adapter location.`,
-      )
+      throw new AdapterConfigError({
```

Comment:
can we also add that the node op should also check to see if the underlying source EA is also working?  There have been times where the issue is not that the composite EA points to the wrong location but that the source EA itself is having issues.